Clarification: API access (#38)
vippsas/vipps-report-api: vipps-report-api-faq.md

Currently we only support single-MSN API keys (see the question below
on API keys for more information on future plans for partner keys).

## What are the benefits of the Report API over the SFTP service?

The
[Vipps SFTP Report Service](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/settlements/sftp-report-service)
lets merchants and partners download settlement files by SFTP.

The Report API has some benefits over this:
* Data can be retrieved at any time.
* Data is available regardless of whether the balance is positive or negative,

>   so even if there has only been refunds there will be data available.

Reviewer: ```suggestion
  so even if there have only been refunds, there will be data available.
```

* The SFTP service only provides settlement files if the balance is positive.

> * It is possible to retrieve data for more than one day in one request, so

Reviewer: I didn't like using "it is possible" or "in one request" twice in the same sentence. You could perhaps say: "It is possible to retrieve data for more than one day in a request, so you could, for example, retrieve an entire weekend, week, or month."

Author: Thanks, fixed!

  it is possible to retrieve an entire weekend, week, month, etc. in one request.
* The SFTP service only provides one file per day.

> * The data is in JSON format, making it easy to use it in many different ways.

Reviewer: ```suggestion
* The data is in JSON format, making it easy to use in many different ways.
```

an amount is always withheld to compensate for high risk, etc. Due to this, each
payment may not always be connected to a specific payout.

## How can I get access to the API?

The API is available for sale units that already have access to a
[Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).

> If you do not already have API access, you can not get access to the Report API.

Reviewer: ```suggestion
If you do not already have API access, you cannot get access to the Report API.
```

This is the case if you only have a "Vippsnummer" sale unit, since those
sale units have no API access.

## Which API keys give access to the API?

For now: The following API keys give access to the Report API:

* The merchant's own API keys: The same API keys that are used to make payments, etc. See

>   [Vipps APIs]([Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).

Reviewer: Maybe you want to use the API keys URL instead?
https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/api-keys

Author: Good idea!

Later (we can not give an ETA, but will update here):

* [Partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys):

>   The same API keys that are used by partner to make payments on behalf of their merchants.

Reviewer: ```suggestion
  The API keys that are used by partners to make payments on behalf of their merchants.
```

The merchant's own API keys give full access to the Report API.

If the merchant uses an integration partner (see

> [partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),

Reviewer: ```suggestion
[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types)),
```

Reviewer: You can't add that slash there, Christian. It is one of the ones that gives a 404 :-0

also give access to the Report API.

If the merchant has a sale unit that is configured with a

> [platform partner](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types),

Reviewer: same slash problem as above.
